app/services/user/create.rb

```diff
@@ -0,0 +1,20 @@
+class User::Create < ApplicationService
+  def initialize(params)
+    @user = params[:user]
+  end
+
+  def call
+    create_profile(@user) if save_user(@user)
+    @user
+  end
+
+  protected
+
+  def save_user(user)
```

Comment:
I would not create this method. `@user.save`is quite short (shorter then your method name) and easy to read and follow